Module_8/index.html

```diff
@@ -0,0 +1,8 @@
+<html>
+    <head>
+    </head>
+    <body>
+        <h1>Assignments</h1>
+        <script src="Q6.js"></script>
+    </body>
+</html>
```

Comment:
Great. All looks good. Lets test how much perfect you have learned from this assignment. Here is an optional code assignment from me. You can complete whenever you have time

Given an array of asynchronous tasks represented by promises (such as chopping, baking, serving, and eating), you have implemented a function executeTasks that executes these tasks sequentially and returns an array of responses when all tasks are completed

const tasks = [
  function() {
    return new Promise((resolve) => {
      setTimeout(() => {
        resolve("Chopping done");
      }, 1000);
    });
  },
  function() {
    return new Promise((resolve) => {
      setTimeout(() => {
        resolve("Baking done");
      }, 2000);
    });
  },
  function() {
    return new Promise((resolve) => {
      setTimeout(() => {
        resolve("Serving done");
      }, 1000);
    });
  },
  function() {
    return new Promise((resolve) => {
      setTimeout(() => {
        resolve("Eating done");
      }, 1000);
    });
  }
];

// Output after 5 seconds - ["Chopping done", "Baking done", "Serving done", "Eating done"]

Hint - Don't use chatgpt for help. Just give it a try